api/controllers/v1/jobs.ts

```diff
@@ -23,12 +23,14 @@ export const TriggerLocalBuild = async (event: any = {}, context: any = {}): Pro
     consoleLogger.info(body.jobId, 'enqueuing Job');
     await sqs.sendMessage(new JobQueueMessage(body.jobId, JobStatus.inQueue), c.get('jobUpdatesQueueUrl'), 0);
     consoleLogger.info(body.jobId, 'Job Queued Job');
+    client.close();
```

Comment:
do we explicitly need to call `client.connect()` in these cases? we are just getting db info and the promise calls to insert can throw namespace errors later if they don't exist. CMIIW

i've been noticing on [node driver examples](https://www.mongodb.com/docs/drivers/node/v5.1/usage-examples/insertOne/) that they do not explicitly call `client.connect()` and the node source code shows that there is a check for connection with [async executions](https://github.com/mongodb/node-mongodb-native/blob/v5.1.0/src/operations/execute_operation.ts#L82)

that said, above suggestion will require some more testing that nothing breaks, so i am fine with at least explicitly closing connection

Reply:
Very good point - I don't think we actually need to the more I think about it. But I also think that with the V2 set of lambdas removing the strict dependency on insertion into the `queue` collection for job execution, it's a bit of a moot point - and I'd rather get this fix incorporated until V2 is used for content staging.